galaxy/importer/loaders/role.py

```diff
@@ -291,7 +291,7 @@ def _load_container_yml(self):
         ansible_container_meta_file = os.path.join(
             self.path, self.ANSIBLE_CONTAINER_META_FILE)
         if os.path.exists(ansible_container_meta_file):
-            if self._container_yml_type is not None:
+            if container_yml_type is not None:
                 raise exc.ContentLoadError(
```

Comment:
Not directly related to this pr, but just noticed exc.ContentLoaderError is a subclass of 'ImportError' which seems wrong. Should it be a subclass of exc.ImporterError instead?

Reply:
Thanks for pointing this out. You're right, will fix it.